### PR TITLE
Switch from requests to axios

### PR DESCRIPTION
**Pull request by PineappleIOnic**
+ Change request library from Requests to Axios

This is a PR in accordance to #33

### CODE REVIEW

File: templates/node/lib/client.js.twig [hunk 1]
```diff
@@ -70,23 +71,39 @@ class Client {
...
 
         let contentType = headers['content-type'].toLowerCase();
+
+        let formData = null;
+
+        // Compute FormData for axios and appwrite.
+        if (contentType.startsWith('multipart/form-data')) {
+            const form = new FormData();
+            
+            let test = this.flatten(params);
```
Comment (eldadfux) [inline templates/node/lib/client.js.twig:81]: Probably better use a more meaningful name here

File: templates/node/lib/client.js.twig [hunk 2]
```diff
@@ -70,23 +71,39 @@ class Client {
...
+            
+            for (const key in test) {
+                form.append(key, test[key])
```
Comment (eldadfux) [inline templates/node/lib/client.js.twig:84]: same here obviously.

File: templates/node/lib/client.js.twig (hunk 1 above)
Comment (PineappleIOnic) [reply templates/node/lib/client.js.twig:81]: Oh, yeah. I forgot about that, that's when I was trying to figure it out by quickly prototyping new ideas

Comment (PineappleIOnic) [reply templates/node/lib/client.js.twig:81]: I'll fix it now

**Pull request merged**